styles/styles.less

```diff
@@ -1,5 +1,9 @@
 @import "variables";
 
+atom-pane-container atom-pane div.item-views {
```

Comment:
Hmm, not sure about this, I fear there might be some side-effects depending on how themes handle panes content. What is this supposed to fix? Maybe there's a better way.